### PR TITLE
Update how to use all-contributors CLI

### PR DESCRIPTION
**Pull request by kachick**
https://github.com/mobu-of-the-world/mobu/pull/1120#discussion_r1342035602

The usage maybe changed 🤔 

https://github.com/all-contributors/all-contributors/blob/67d2fb18f6bea131add3141551c23999baa85264/docs/cli/installation.md#L17-L18

### CODE REVIEW

**Review by ujihisa — APPROVED** [review]

File: .all-contributorsrc [hunk 1]
```diff
@@ -96,15 +96,6 @@
...
-    },
-    {
-      "login": "matz",
-      "name": "Yukihiro \"Matz\" Matsumoto",
-      "avatar_url": "https://avatars.githubusercontent.com/u/30733?v=4",
-      "profile": "https://github.com/matz",
-      "contributions": [
-        "code"
-      ]
     }
```
Comment (ujihisa) [inline .all-contributorsrc:99]: :eyes:

**Review by ujihisa — APPROVED** [review]

File: .all-contributorsrc (hunk 1 above)
Comment (kachick) [reply .all-contributorsrc:99]: Accidentally included local testing commit 😋 

(Sorry matz)

**Comment by kachick** [comment]
Thanks for your review!

**Comment by ujihisa** [comment]
:+1:

**Pull request merged**